services/navet-ms/helpers/parser.js

```diff
@@ -1,4 +1,5 @@
 import parser from 'xml2js';
+import { ResourceNotFoundError } from '@helsingborg-stad/npm-api-error-handling';
 
 export const parseXml =
```

Comment:
While reading through this for the first time I found `parseXml` and `parseJSON` to be a bit misleading. `parseXml` doesn't actually parse any XML, it takes an object and constructs a specific XML payload. Maybe something like `makePersonpostRequestXML` would be more appropriate?

`parseJSON` doesn't parse JSON either, it takes XML and parses it into a JSON object. A bit more unsure about this one, maybe `parseToJSON` or `parseXML`? Considering the inconsistencies with the 3rd party function `parser.parseString` following the `parseX` convention where X denotes the input type. On the other hand, functions like `parseInt` follow a different pattern where X denotes the output type, but I'm mentioning it since I found it confusing at first.

These are just some side-suggestions while we're editing this file anyway 🙂.